src/app/app-routing.module.ts

```diff
@@ -32,7 +32,7 @@ const appRoutes: Routes = [
 ];
 
 @NgModule({
-  imports: [RouterModule.forRoot(appRoutes)],
+  imports: [RouterModule.forRoot(appRoutes, { relativeLinkResolution: 'legacy' })],
```

Comment:
It is possible to migrate the application to the default value `corrected `?
https://angular.io/api/router/ExtraOptions